### ISSUE TITLE
Placeholder of community feature requests

### ISSUE THREAD

**Issue by pietrushnic** [Dasharo/dasharo-issues#71]
**The problem you're addressing (if any)**

Various community needs

**Describe the solution you'd like**

We should split that into consumable requirements. Right now I'm just copying from Phornix Community member answer. This is by Termy:
> I think things like the different voltages, multiplier and baseclocks, power limits are expected for the CPU. I don't know if tuning individual cores or things like that are really too broadly used outside of benchmark-hunters though.
What i would deem even more important nowadays are comprehensive options for RAM-OC as this can yield much more of a performance boost (as nowadays the binning is much more strict and most higher-end CPUs are already running close to their limit). I'm not too familiar with the intricacies of Alder Lake and DDR5 yet, but detailed settings for the different (sub-)timings, voltages (both for the ram and the cpu-side), synchronous or asynchronous operation (at least i think AL does have a similar setting for the ram-controller as AMDs infinity fabric clock?) are the things i would look out for. And of course - as already mentioned in other posts - ECC support would be nice as well ^^

**Where is the value to a user, and who might that user be?**

Value is better performance per USD.

**Describe alternatives you've considered**

**Additional context**

Phoronix Community is well respected group of open-source enthusiasts. Their suggestions should be indication of market demand.

**Comment by pietrushnic** [comment]
Support for MSI PRO B660M-A WIFI DDR4:
https://twitter.com/MartynHare/status/1528081631747219456

**Comment by Firminator** [comment]
I'd argue that adding RAM or CPU overclocking/timing features in Dasharo/Coreboot GUI is shooting way beyond the goal while adding XMP is not.

Also I'd personally like to disable every device I'm not using in stock BIOS ever since I started building PCs in the late 90s. In other words for the MSI Z690 I'd like to see the possibility in Dasharo to disable the AHCI/SATA controller and/or NVMe controller (depending what type of SSD we are using). Also disabling unused USB ports would be nice.

**Comment by pietrushnic** [comment]
@Firminator we discussed that yesterday during vPub and this is definitely something that should go to our backlog. We added some notes to vPub Etherpad (available from [Dasharo Matrix](https://matrix.to/#/#dasharo:matrix.org) widget).

**Comment by Firminator** [comment]
Yes I saw the discussion in vPub. Glad to hear this is being considered.
